app/services/induction/enrol.rb

```diff
@@ -2,15 +2,21 @@
 
 class Induction::Enrol < BaseService
   def call
-    participant_profile.induction_records.create!(
-      induction_programme: induction_programme,
-      start_date: start_date,
-      induction_status: :active,
-      schedule: participant_profile.schedule,
-      preferred_identity: preferred_identity,
-      mentor_profile: mentor_profile,
-      school_transfer: school_transfer,
-    )
+    ActiveRecord::Base.transaction do
+      participant_profile.induction_records.create!(
+        induction_programme: induction_programme,
+        start_date: start_date,
+        training_status: :active,
+        induction_status: :active,
+        schedule: participant_profile.schedule,
+        preferred_identity: preferred_identity,
+        mentor_profile: mentor_profile,
+        school_transfer: school_transfer,
+      )
+      # TODO: here would be a convenient place to reactivate a withdrawn / deferred profile
+      # However this needs some checking/work by the LP team
+      # reactivate_profile! unless participant_profile.active_training_status?
+    end
```

Comment:
I think the best plan is for us to remove the validation so that we don't have to check if there is an active state on the participant_profile and we just allow it